src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java

```diff
@@ -414,6 +414,10 @@ public int compareTo(Tag object) {
             return result;
         }
 
+        /**
+         * Indicates whether some other object is "equal to" this one.
+         * @noinspection EqualsCalledOnEnumConstant
```

Comment:
reason of suppression to javdoc descrition

Reply:
Done.